eng/common/TestResources/TestResources-Helpers.ps1

```diff
@@ -129,8 +129,30 @@ function MergeHashes([hashtable] $source, [psvariable] $dest) {
     }
 }
 
+function IsBicepInstalled() {
+    try {
+        bicep --version | Out-Null
+        return $LASTEXITCODE -eq 0
+    }
+    catch {
+        return $false
+    }
+}
+
+function IsAzCliBicepInstalled() {
+    try {
+        az bicep version | Out-Null
```

Comment:
Is this extension installed by default with az?

Reply:
no. The version check is required to see if `az bicep install` has been called